.pre-commit-config.yaml

```
@@ -108,7 +108,7 @@ repos:
     language_version: python3
 
 - repo: https://github.com/PyCQA/pylint.git
-  rev: v2.15.9
+  rev: v3.0.0a6
```

Let's revert this for now:
```suggestion
  rev: v2.15.9
```